core/modules/filters.js

```diff
@@ -30,7 +30,8 @@ function parseFilterOperation(operators,filterString,p) {
 		operator = {};
 		// Check for an operator prefix
 		if(filterString.charAt(p) === "!") {
-			operator.prefix = filterString.charAt(p++);
+			operator.prefix = filterString.charAt(p++); // remove?
```

Comment:
but this is really about fixing negation and depends on that construct. I would rather not make a pull request with that function first only to use it here.